### PR TITLE
Added page for the Social

### PR DESCRIPTION
**Pull request by jberkus**
Also added links and information for the unconference.

Attn: @puja108 @pnbrown @mrbobbytables @sftim

### CODE REVIEW

File: content/en/events/kcseu/_index.md [hunk 1]
```diff
@@ -21,14 +21,12 @@ Similar to North America summit last year, the summit in Europe will be
 informal with minimal programming, but plenty of space to hang out, catch up,
 and meet new contributors.
 
-We are looking for SIG meetings, session leaders for unconference sessions, plus 
-one or more pre-planned plenary sessions.  If you have an idea for a session, please
-look for our thread on session ideas coming soon. If your SIG wants to use the 
-Contributor Summit to have an in-person work session
-or meeting, please [sign up](https://docs.google.com/spreadsheets/d/1LS1q4VJajZkNZODdMnJ4eH3XnTsE_4em6pjQgToHyfw/edit?usp=sharing).
+We are looking for SIG meetings, [session leaders for unconference sessions](https://github.com/kubernetes/community/issues/6633), plus
```
Comment (puja108) [inline content/en/events/kcseu/_index.md:24]:
```suggestion
We are looking for [SIG meetings](https://docs.google.com/spreadsheets/d/1LS1q4VJajZkNZODdMnJ4eH3XnTsE_4em6pjQgToHyfw/edit?usp=sharing), [session topics and leaders for unconference sessions](https://github.com/kubernetes/community/issues/6633), plus 
```

or did you want to remove the SIG signup sheet on purpose?

File: content/en/events/kcseu/_index.md [hunk 2]
```diff
@@ -21,14 +21,12 @@ Similar to North America summit last year, the summit in Europe will be
...
-Contributor Summit to have an in-person work session
-or meeting, please [sign up](https://docs.google.com/spreadsheets/d/1LS1q4VJajZkNZODdMnJ4eH3XnTsE_4em6pjQgToHyfw/edit?usp=sharing).
+We are looking for SIG meetings, [session leaders for unconference sessions](https://github.com/kubernetes/community/issues/6633), plus 
+one or more pre-planned plenary sessions.  If you have an idea for a session, or 
+if your SIG wants to use the Contributor Summit to have an in-person work session
+or meeting, please [email us].
 
-We'll start the day with a Steering Committee AMA. There will also be an all-day
-documentation sprint, led by SIG-Docs.
+There will also be a documentation sprint, led by SIG-Docs.
```
Comment (sftim) [inline content/en/events/kcseu/_index.md:29]: nit:
```suggestion
There will also be a documentation sprint, led by SIG Docs.
```

File: content/en/events/kcseu/_index.md (hunk 1 above)
Comment (jberkus) [reply content/en/events/kcseu/_index.md:24]: thanks, change made

File: content/en/events/kcseu/_index.md (hunk 2 above)
Comment (jberkus) [reply content/en/events/kcseu/_index.md]: fixed

**Review by puja108 — APPROVED** [review]

**Pull request merged**